ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxMain.scala

```diff
@@ -5,16 +5,31 @@ package com.digitalasset.platform.sandbox
 
 import java.util.concurrent.atomic.AtomicBoolean
 
+import ch.qos.logback.classic.Level
+import org.slf4j.Logger
 import com.digitalasset.platform.sandbox.cli.Cli
 import org.slf4j.LoggerFactory
 
 import scala.util.control.NonFatal
 
 object SandboxMain extends App {
 
+  // Copied from language-support/scala/codegen/src/main/scala/com/digitalasset/codegen/Main.scala
+  private def setGlobalLogLevel(verbosity: Level): Unit = {
+    LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME) match {
+      case a: ch.qos.logback.classic.Logger =>
```

Comment:
nit-pick: why `a` instead of `logger`?

Reply:
Named it `rootLogger` to distinguish with this class logger.